### DISCUSSION TITLE
Discussion Place

### DISCUSSION THREAD

**Discussion by amir-the-h** [go-kit/kit#1179] [Q&A]
Hi there!

I wonder now when the slack join link is not valid and also I can't find the IRC channel details, is there any way to join the go-kit community?

**Comment by souvikhaldar** [comment]
There is a #go-kit channel on gophers slack.

**Reply by amir-the-h** [reply]
But workspace needs invitations. And the invitation link is no longer available.

**Reply by ChrisHines** [reply]
I think the current invite link is https://invite.slack.golangbridge.org/
